pom.xml

```diff
@@ -46,7 +46,7 @@
         <!-- Pick up common dependencies for the selected LTS line: https://github.com/jenkinsci/bom#usage -->
         <groupId>io.jenkins.tools.bom</groupId>
         <artifactId>bom-${jenkins.baseline}.x</artifactId>
-        <version>4136.vca_c3202a_7fd1</version>
+        <version>4948.vcf1d17350668</version>
```

Comment:
version we had in bom had an old version of pipeline-input-step which didn't allow the custom cancel text